README.md

```diff
@@ -68,6 +68,19 @@ func open(sender: AnyObject) {
 }
 ```
 
+You can also use your own FolioReader view controller like this.
+
+```swift
+		let config = FolioReaderConfig()
+
+		let bookPath = NSBundle.mainBundle().pathForResource("book", ofType: "epub")
+		let epubVC = FolioReaderContainer(config: config, epubPath: bookPath, removeEpub: true)
```

Comment:
This init still not public yet, we have to make it public on `FolioReaderContainer`.

```
public init(config configOrNil: FolioReaderConfig!, epubPath epubPathOrNil: String? = nil, removeEpub: Bool) {
```